Products/CMFPlone/recyclebin.py

```diff
@@ -125,6 +125,9 @@ def get_items_sorted_by_date(self, reverse=True):
 class RecycleBin:
     """Stores deleted content items"""
 
+    # Permission for managing recycle bin
+    MANAGE_RECYCLEBIN = "Products.CMFPlone.ManageRecycleBin"
```

Comment:
This should be "Manage recycle bin". `checkPermission` uses the permission's title, not its id

Reply:
done